mergin/client.py

```diff
@@ -165,6 +165,9 @@ def inspect_files(self):
             for file in files:
                 if self.ignore_file(file):
                     continue
+                if "gpkg" in file:
```

Comment:
I think here is too early to decide whether we want to commit changes from WAL or we do geodiff